sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveResolutionSuite.scala

```diff
@@ -81,9 +81,13 @@ class HiveResolutionSuite extends HiveComparisonTest {
       .toDF().registerTempTable("caseSensitivityTest")
 
     val query = sql("SELECT a, b, A, B, n.a, n.b, n.A, n.B FROM caseSensitivityTest")
-    assert(query.schema.fields.map(_.name) === Seq("a", "b", "A", "B", "a", "b", "A", "B"),
+    assert(query.schema.fields.map(_.name) === Seq("a", "b", "a", "b", "a", "b", "a", "b"),
       "The output schema did not preserve the case of the query.")
```

Comment:
Supporting normalization is good. However, when explicitly specifying the case in the query, should we need to preserve the case of the query, instead of normalizing it like this?